repos/fdbt-site/src/pages/capConfirmation.tsx

```diff
@@ -0,0 +1,229 @@
+    tapsPricingContents: string[];
+    capDistancePricingContents: string[];
+    csrfToken: string;
+}
+
+export const buildCapConfirmationElements = (
+    typeOfCap: string,
+    productGroupName: string,
+    caps: Cap[],
+    capValidity: string,
+    capStartInfo: CapStartInfo,
+    services: string[],
+    tapsPricingContents: string[],
+    capDistancePricingContents: string[],
+): ConfirmationElement[] => {
+    const confirmationElements: ConfirmationElement[] = [
+        {
+            name: 'Cap type',
+            content: `Pricing by ${typeOfCap.substring(2)} `,
+            href: 'typeOfCap',
+        },
+    ];
+
+    if (productGroupName) {
+        confirmationElements.push({
+            name: 'Product Group Name',
+            content: productGroupName,
+            href: '/selectCappedProductGroup',
+        });
+    }
+
+    caps.forEach((cap) => {
+        const durationText = cap.durationUnits
+            ? `${cap.durationAmount} ${cap.durationUnits}${cap.durationAmount === '1' ? '' : 's'}`
+            : `${cap.durationAmount}`;
+
+        confirmationElements.push({
+            name: cap.name,
+            content: [`Price - £${cap.price}`, `Duration - ${durationText}`],
+            href: '/createCaps',
+        });
+    });
+
+    confirmationElements.push({
+        name: 'Cap Expiry',
+        content: sentenceCaseString(capValidity),
+        href: '/selectCapValidity',
+    });
+
+    confirmationElements.push({
+        name: 'Cap Starts',
+        content:
+            capStartInfo.type === 'fixedWeekdays' && capStartInfo.startDay
+                ? `Fixed Days - ${sentenceCaseString(capStartInfo.startDay)}`
+                : 'Rolling Days',
+        href: '/defineCapStart',
+    });
+
+    if (services.length > 0) {
+        confirmationElements.push({
+            name: 'Services',
+            content: services.join(', '),
+            href: '/serviceList',
+        });
+    }
+
+    if (tapsPricingContents.length > 0) {
+        confirmationElements.push({
+            name: 'Prices by Taps',
+            content: tapsPricingContents,
+            href: '/multiTapsPricing',
+        });
+    }
+
+    if (capDistancePricingContents.length > 0) {
+        confirmationElements.push({
+            name: 'Prices',
+            content: capDistancePricingContents,
+            href: '/defineCapPricingPerDistance',
+        });
+    }
+
+    return confirmationElements;
+};
+
+const CapConfirmation = ({
+    typeOfCap,
+    productGroupName,
+    caps,
+    capValidity,
+    capStartInfo,
+    services,
+    tapsPricingContents,
+    capDistancePricingContents,
+    csrfToken,
+}: CapConfirmationProps): ReactElement => (
+    <TwoThirdsLayout title={title} description={description} errors={[]}>
+        <CsrfForm action="/api/capConfirmation" method="post" csrfToken={csrfToken}>
+            <>
+                <h1 className="govuk-heading-l">Check your answers before sending your fares information</h1>
```

Comment:
```suggestion
                <h1 className="govuk-heading-l">Check your answers before submitting your cap(s) information</h1>
```

Could you please also change the h1 in faresConfirmation.tsx, from

 <h1 className="govuk-heading-l">Check your answers before sending your fares information</h1>

to

 <h1 className="govuk-heading-l">Check your answers before submitting your fares information</h1>